requirements/base.in

```diff
@@ -7,7 +7,7 @@ django-model-utils
 djangorestframework
 django-ipware>=1.1.0
 jsonfield2
-pyjwt<2.0.0 # Used for encoding JWTs when communicating with proctoring provider backends
+pyjwt		# Used for encoding JWTs when communicating with proctoring provider backends
```

Comment:
We need to add a minimum version here since we are bound to the newer version. See this example constraint and related comment: https://github.com/edx/edx-drf-extensions/blob/master/requirements/base.in#L10